editor/src/components/canvas/controls/select-mode/select-mode-hooks.tsx

```diff
@@ -175,11 +175,10 @@ export function getSelectableViews(
 
     const selectableViewsFiltered = uniqueSelectableViews.filter((view) => {
       // I kept the group-like behavior here that the user can't single-click select the parent group, even though it is a view now
-      const isGroup = MetadataUtils.isAutoSizingViewFromComponents(componentMetadata, view)
       const isAncestorOfSelected = selectedViews.some((selectedView) =>
         EP.isDescendantOf(selectedView, view),
       )
-      if (isGroup && isAncestorOfSelected) {
```

Comment:
This looks wrong. It looks like this `if` statement should go in the bin now as it would have previously always been `false` (so the `else` clause would execute, returning `true`)

Reply:
That works for me, I _think_ at the point when I looked at it I was concerned that I would be removing prior (to the group condition) behaviour.

Reply:
Fixed in 7585ad850669034f876221620ecc02d4879ec51c.